Engineering-Reference/GoogleDestinationAO.md

```diff
@@ -454,31 +438,31 @@ Default Value: On
 
 Check that users/resources/groups/shared drives exist within the destination system.
 
-Change Conditions: ??
+Change Conditions: When disabled a check will not be performed before the migration.
 
 ### Check Services Enabled <a name="cheseren"></a>
 {: .no_toc }
 Default Value: Off
 
 When running the 'Check Users' action, also check if the selected Google services for that user are enabled.
 
-Change Conditions: ??
+Change Conditions: When enabled an additional check will be performed to check the Google services (Gmail, Drive etc.) are enabled for the users.
```

Comment:
Where does this check happen ?  Where are respected results ?  what does this do to the migration ? if they are not enabled for the user does it enable it?

Reply:
Will clarify